test/functional/http/get/peers.js

```diff
@@ -1,494 +1,157 @@
-							if (res.body.peers[i + 1] != null) {
-								node.expect(res.body.peers[i + 1].state).to.be.at.most(res.body.peers[i].state);
-							}
-						}
-					}
-
-					done();
-				});
-			});
-
-			it('using sort with any of sort fields should not place NULLs first', function (done) {
-				node.async.each(peersSortFields, function (sortField, cb) {
-					http.get('/api/peers?orderBy=' + sortField, function (err, res) {
-						node.expect(res.body).to.have.property('peers').that.is.an('array');
-
-						var dividedIndices = res.body.peers.reduce(function (memo, peer, index) {
-							memo[peer[sortField] == null ? 'nullIndices' : 'notNullIndices'].push(index);
-							return memo;
-						}, {notNullIndices: [], nullIndices: []});
-
-						if (dividedIndices.nullIndices.length && dividedIndices.notNullIndices.length) {
-							var ascOrder = function (a, b) {
-								return a - b;
-							};
-							dividedIndices.notNullIndices.sort(ascOrder);
-							dividedIndices.nullIndices.sort(ascOrder);
-
-							node.expect(dividedIndices.notNullIndices[dividedIndices.notNullIndices.length - 1])
-								.to.be.at.most(dividedIndices.nullIndices[0]);
-						}
-						cb();
-					});
-				}, function () {
-					done();
+		it('using valid broadhash = "198f2b61a8eb95fbeed58b8216780b68f697f26b849acf00c8c93bb9b24f783d" should return the result', function () {
+			return peersEndpoint.makeRequest({broadhash: validHeaders.broadhash}, 200)
+				.then(function (res) {
+					res.body.peers[0].broadhash.should.be.eql(validHeaders.broadhash);
 				});
-			});
 		});
 
-		describe('limit', function () {
-
-			it('using string limit should fail', function (done) {
-				var limit = 'one';
-				var params = 'limit=' + limit;
-
-				http.get('/api/peers?' + params, function (err, res) {
-					node.expect(res.body).to.have.property('message').to.equal('Expected type integer but found type string');
-					done();
-				});
-			});
-
-			it('using limit = -1 should fail', function (done) {
-				var limit = -1;
-				var params = 'limit=' + limit;
-
-				http.get('/api/peers?' + params, function (err, res) {
-					node.expect(res.body).to.have.property('message').to.equal('Value -1 is less than minimum 1');
-					done();
-				});
-			});
-
-			it('using limit = 0 should fail', function (done) {
-				var limit = 0;
-				var params = 'limit=' + limit;
-
-				http.get('/api/peers?' + params, function (err, res) {
-					node.expect(res.body).to.have.property('message').to.equal('Value 0 is less than minimum 1');
-					done();
-				});
-			});
-
-			it('using limit = 1 should be ok', function (done) {
-				var limit = 1;
-				var params = 'limit=' + limit;
-
-				http.get('/api/peers?' + params, function (err, res) {
-					node.expect(res.body).to.have.property('peers').that.is.an('array');
-					node.expect(res.body.peers.length).to.be.at.most(limit);
-					done();
+		it('using sort = "version:asc" should be ok', function () {
```

Comment:
Change `should be ok` to `should return results in ascending / descending` order.

Reply:
Done.